iopipe/iopipe.py

```diff
@@ -223,10 +230,7 @@ def send(self, context=None):
         self.report['time_nanosec'] = \
             int((time.time() - self._time_start) * 1000000000)
 
-        # Falsify function_id
-        self.report['function_id'] = '0xDEADBEEF'
```

Comment:
We don't use it anywhere. It's not even a real value here with Python. I've dropped it from Node where it was doing sha256 operations (and since we didn't use it... it was unnecessary bloat)